navbar.css

```diff
@@ -0,0 +1,68 @@
+nav {
+    background: #ffffff;
+    opacity: 0.5;
+    width: 100%;
+    height: 75px;
+    position: fixed;
+    top: 0;
+    z-index: 1;
+}
+
+.menu {
+    display: table;
```

Comment:
ewwwww, try and avoid this and use a different approach 
in this instance maybe set a width so the margin: 0 auto; works for you